libcontainer/intelrdt/stats.go

```diff
@@ -45,6 +45,9 @@ type Stats struct {
 	// The memory bandwidth schema in 'container_id' group
 	MemBwSchema string `json:"mem_bw_schema,omitempty"`
 
+	// The full schemata in 'container_id' group
```

Comment:
nit: please write full sentences, with a period (even if adjacent ones don't have it).

Reply:
Perhaps it also makes sense to note that the value here may overwrite L3CacheSchema and MemBwSchema?